app/models/enrollment.rb

```diff
@@ -0,0 +1,4 @@
+class Enrollment < ApplicationRecord
```

Comment:
@fpgentil, What u think about `Enrollment`? 
Which one do u think that is better? `Enrollment` or `Participation`?